core/templates/dev/head/pages/exploration_player/ExplorationPlayer.ts

```diff
@@ -62,3 +62,19 @@ require('pages/exploration_player/ConversationSkinDirective.ts');
 require('pages/exploration_player/ExplorationFooterDirective.ts');
 require('pages/exploration_player/LearnerLocalNav.ts');
 require('pages/exploration_player/LearnerViewInfo.ts');
+
+require('services/ContextService.ts');
+require('services/PageTitleService.ts');
+require('domain/exploration/ReadOnlyExplorationBackendApiService.ts');
+
+oppia.controller('ExplorationPlayer', [
+  'ContextService', 'PageTitleService', 'ReadOnlyExplorationBackendApiService',
+  function(
+      ContextService, PageTitleService, ReadOnlyExplorationBackendApiService) {
+    var explorationId = ContextService.getExplorationId();
+    ReadOnlyExplorationBackendApiService.fetchExploration(explorationId, null)
```

Comment:
If it's fetching the entire exploration from the server just to use the title, can we find a better way?

Reply:
Not really, but the exploration being fetched can be used later to remove the GLOBALS used in exploration_player.html. I am planning on using it during the GSOC period to remove GLOBALS from the page.